libs/cognitive-extension/application/src/impl/execute_tool_handler_impl.py

```
@@ -0,0 +1,36 @@
+        try:
+            entity = ToolCall(
+                id=str(uuid.uuid4()),
+                tool_name=command.tool_name,
+                parameters=command.parameters,
+                result={},
+                status="pending",
+                created_at=datetime.now(timezone.utc),
+            )
+            await self._repository.save(entity)
+            return ExecuteToolResult(success=True, result=entity)
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)